docs/netdata-cloud-onprem/getting-started.md

```diff
@@ -0,0 +1,20 @@
+# Getting started with Netdata Cloud On-Prem
+Helm chart for the Netdata Cloud On-Prem installation on Kubernetes is available at ECR registry.
+ECR registry is private, so you need to login first. Credentials are sent by our Marketing team. If you do not have them, please contact our marketing team - marketing@netdata.cloud.
```

Comment:
i think we should use a better email here marketing@ does not seem right for this. @netdata/product thoughts?

Reply:
I agree